_episodes/03-github.md

```diff
@@ -56,8 +56,7 @@ You should see no output.
 Now, follow the instructions on GitHub under "...or push an existing repository from the command line"
 ~~~
 $ git remote add origin git@github.com:YOUR_GITHUB_USERNAME/molecool.git
-$ git branch -M main
-$ git push -u origin main
+$ git branch
```

Comment:
I think we will want to leave original instructions here to match instructions on GitHub. If your branch is already named `main`, the command does nothing, so there is no harm in leaving it in.